entities/storagestate/status_test.go

```diff
@@ -24,6 +24,7 @@ func TestStatusValidation(t *testing.T) {
 			"read only",
 			"ok",
 			"WRITEONLY",
+			"INDESKING",
```

Comment:
Maybe INDEXING instead of INDESKING?

Reply:
That's actually the failing test. It's to make sure typos don't accidentally generate valid states